README.md

```diff
@@ -40,5 +45,15 @@
   > bundle:start <net.staticsnow:nexus-repository-apt ID>
   ```
 
+#### Permanent Install
+
+For more permanent installs of the nexus-repository-apt plugin, follow these instructions:
+
+* Copy the bundle (nexus-repository-apt-1.0.2.jar) into <nexus_dir>/deploy
```

Comment:
I don't know if it implies, since you need to do all of this yourself :)

I definitely agree we need a better approach to adding these.